dist/uiUpdater.js

```diff
@@ -115,10 +115,11 @@ function renameProject(project) {
     let projectTitle = document.querySelector(`.${project.name.replace(/\s/g, '').toLowerCase()}Title`);
     projectTitle.replaceWith(document.createElement('input'));
     let inputValue = document.querySelector('input');
+    console.log(inputValue);
     inputValue === null || inputValue === void 0 ? void 0 : inputValue.addEventListener('keypress', function (event) {
         if (event.key === 'Enter') {
-            if (checkNameProject(inputValue === null || inputValue === void 0 ? void 0 : inputValue.value) === true) {
-                alert("Projects can't be named the same");
+            if (checkNameProject(inputValue === null || inputValue === void 0 ? void 0 : inputValue.value) === true || (inputValue === null || inputValue === void 0 ? void 0 : inputValue.value.length) === 0) {
```

Comment:
do you need the `inputValue === null || inputValue === void 0 ? void 0 :` here? can leave just the .length check?

Reply:
dist